site/content/docs/5.3/components/side-navigation.md

```diff
@@ -0,0 +1,1425 @@
+---
+layout: docs
+title: Side Navigation
+description: Documentation and examples for Boosted's exclusive Brand responsive side navigation.
+group: components
+aliases:
+  - "/docs/components/side-navigation/"
+toc: true
+added: "5.3"
+---
+
+## How it works
+
+Our side navigation is basically a `<nav>` that can provide up to 3 levels hierarchy between links.
+
+Side navigation should be used inside an offcanvas using `.offcanvas-*` for responsive purpose. By default, we use `.offcanvas-lg` in our docs. Please refer to [Offcanvas]({{< docsref "/components/offcanvas#responsive" >}}) to learn more about its behavior. Since it uses [responsive offcanvas]({{< docsref "/components/offcanvas#responsive" >}}), please resize your browser to see its responsive behavior in action.
+
+{{< callout warning >}}
+In order to make side navigation work properly, please make sure that the Javascript files from [Collapse]({{< docsref "/components/collapse#usage" >}}) and [Tabs]({{< docsref "/components/navs-tabs#javascript-behavior" >}}) are imported in your project.
+{{< /callout >}}
+
+The side navigation provides 3 ready-to-use fully responsive themes. For each theme, we provide:
```

Comment:
Not sure about the word 'theme', it sounds like the future 'color themes' and it may be confusing 🤔